x-pack/legacy/plugins/ml/public/components/job_messages/job_messages.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Double comment here. Can we remove one?

Reply:
https://github.com/elastic/kibana/pull/50468/commits/4a1c52faf7fb75555b2358876df521f57573c61f done 👌